examples/recording_service/service_admin/cpp/CMakeLists.txt

```diff
@@ -16,13 +16,26 @@ project (RecorderRemoteAdmin)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CONNEXTDDS_DIR}/resource/cmake")
 
 find_package(
-    RTIConnextDDS "6.0.0"
+    RTIConnextDDS "6.1.0"
     REQUIRED
     COMPONENTS
         core
         messaging_api
+        routing_service
 )
 
+# Select the adequate version of the Recording Service library depending on the
+# build type
+set(recording_lib "rtirecordingservicecore")
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(recording_lib "${recording_lib}d")
+endif()
+
+find_library(
+    recordingservicelib
+    NAMES ${recording_lib}
+    PATHS "${CONNEXTDDS_DIR}/lib/${CONNEXTDDS_ARCH}")
+
```

Comment:
This should go in FindRtiConnextDDS. Perhaps your PR should include the necessary change there, so we don't have to do this.

Reply:
Actually, we should try to get the FindRtiConnextDDS file in this repository, because it'll be more up-to-date.

Reply:
Removed and we are using the FindRtiConnextDDS file from the repository instead of the installation one.